backend/src/main/java/gov/cdc/usds/simplereport/config/SendGridDisabledConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package gov.cdc.usds.simplereport.config;
+
+import com.sendgrid.helpers.mail.Mail;
+import gov.cdc.usds.simplereport.service.email.EmailProviderWrapper;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+@Configuration
+public class SendGridDisabledConfiguration {
```

Comment:
I got distracted an I think this should probably go away, but this is an antipattern (I'm surprised spotless allows it? Or maybe it doesn't?). If you want to have DisabledEmailWrapper be a standalone class, make it an inner static class of SendGridDisabledConfiguration, but since it's a functional interface, you can also just replace the whole thing with

```java
EmailProviderWrapper defaultToBypassSendGrid() {
    return ()->null;
}
```

Reply:
Great catch! used the lambda as you suggested